eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
+
+type RouteList []*Route
+
+// RouteInfo contains a route id, plus the loaded and parsed route or
+// the parse error in case of failure.
+type RouteInfo struct {
```

Comment:
can there be parsing errors in the innkeeper client? in which situations? if the innkeeper response is a single json document, how do identify which individual route's parsing failed? how can unparseable routes get into Innkeeper's storage?